### DISCUSSION TITLE
Get Request Headers

### DISCUSSION THREAD

**Discussion by josejachuf** [salvo-rs/salvo#196] [Q&A] [answered]
Hi. How access to request headers?
I have the Header Authorization and I want to use it in a Middleware.
I saw that there is **parse_headers**, but I don't know how to use it

**Comment by chrislearn** [comment, marked as answer]
https://docs.rs/salvo_core/latest/salvo_core/http/request/struct.Request.html#method.header
https://docs.rs/salvo_core/latest/salvo_core/http/request/struct.Request.html#method.headers
